README.md

```diff
@@ -93,6 +102,20 @@ Ansible Best Practices
   Avoid duplicating the remote full path in the role directory, however, as that creates unnecessary depth in
   the file tree for the role. Grouping sets of similar files into a subdirectory of `templates` is allowable,
   but avoid unnecessary depth to the hierarchy.
```

Comment:
I think a bit more structure is needed. Those items added below were under the "Documentation conventions" header in the working doc.

Reply:
This isn't a change from this PR. This is pre-existing material.

Reply:
I mean the items below.

Reply:
Hmm, maybe move this down to those lines, then. Currently this comment is regarding the structure of a role's `templates` directory based on the PR comment.